modules/gateway/peers.go

```diff
@@ -304,6 +326,19 @@ func acceptableVersion(version string) error {
 	return nil
 }
 
+// acceptableSessionHeader returns an error
+// if the remote header indicates that a connection is impossible.
```

Comment:
Never mind, using Vim, I now see what you mean. I use VSCode for my day-to-day golang programming, only use vim for certain quick edits. Will check if vscode has such an option/plugin available.